conans/test/functional/revisions_test.py

```diff
@@ -88,21 +72,25 @@ def test_update_recipe_iterating_remotes(self):
             other_v2.create(self.ref, conanfile=conanfile)
         other_v2.upload_all(self.ref, remote="remote2")
 
+        remote2_latest_prev = other_v2.get_latest_prev(self.ref)
+
         # Install, it wont resolve the remote2 because it is in the registry, it will use the cache
+        # doing --update it will find a newer same revision in remote2 and update the date and server
+        # of the revision in the local cache
+        # TODO: cache2.0 we get the recipe from remote 2 but package rev still is the one from
+        #  is this what we want?
```

Comment:
Check the TODO comment, is it still valid?

Reply:
I don't think it's valid any more, I'll remove that.